tjdashboard/code/site/models/dashboards.php

```diff
@@ -52,10 +52,8 @@ public function __construct($config = array())
 	 */
 	protected function getListQuery()
 	{
-		$app = JFactory::getApplication();
-
 		// Initialize variables.
-		$db    = JFactory::getDbo();
+		$db    = $this->getDbo();
```

Comment:
Can you just use `$this->_db` ?

Reply:
No, It is protected variable and we can use protected variable in only to it's child class not to the child's child class.